lib/mentat.ex

```diff
@@ -107,22 +112,9 @@ defmodule Mentat do
   """
   @spec get(name(), key()) :: value()
   def get(cache, key) do
-    config = get_config(cache)
-    now    = ms_time(config.clock)
-
-    case :ets.lookup(cache, key) do
-      [] ->
-        :telemetry.execute([:mentat, :get], %{status: :miss}, %{key: key, cache: cache})
-        nil
-
-      [{^key, _val, ts, ttl}] when is_integer(ttl) and ts + ttl <= now ->
-        :telemetry.execute([:mentat, :get], %{status: :miss}, %{key: key, cache: cache})
-        nil
-
-      [{^key, val, _ts, _expire_at}] ->
-        :telemetry.execute([:mentat, :get], %{status: :hit}, %{key: key, cache: cache})
-        val
-    end
+    {status, value} = lookup(cache, key)
+    :telemetry.execute([:mentat, :get], %{status: status}, %{key: key, cache: cache})
```

Comment:
looks like the telemetry could stay in the case statement for lookup/2?

Reply:
CC @GregMefford

Reply:
Aha you're totally right! I'll fix that while it's top of mind before we merge this.